### ISSUE TITLE
Execution failed for task ':app:findLibraries' in demo project

### ISSUE THREAD

**Issue by tkirshboim** [mikepenz/AboutLibraries#488]
I am new to using this great library, so please excuse me if I'm missing something obvious. I thought to report this, since it seems like a bug to me.

## About this issue

- Briefly describe the issue:
`findLibraries` gradle task fails on demo project.

- How can the issue be reproduced:
1. Check out the repository ([this](https://github.com/mikepenz/AboutLibraries/tree/919805c4c94de36cec8ccae0ff96ebd0683e616a)  is the commit I checked out).
2. Run `./gradlew findLibraries`
3. Expected: Works as described [in Readme](https://github.com/mikepenz/AboutLibraries/tree/919805c4c94de36cec8ccae0ff96ebd0683e616a#find).
4. Actual: Show an error
```
./gradlew findLibraries
> Task :app:findLibraries FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:findLibraries'.
> No signature of method: com.mikepenz.aboutlibraries.plugin.AboutLibrariesProcessor.gatherDependencies() is applicable for argument types: (org.gradle.api.internal.project.DefaultProject_Decorated) values: [project ':app']
  Possible solutions: gatherDependencies(java.lang.Object, java.lang.Object)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 630ms
1 actionable task: 1 executed
```

## Details
- Used library, support library, gradle build tools versions: https://github.com/mikepenz/AboutLibraries/tree/919805c4c94de36cec8ccae0ff96ebd0683e616a

**Comment by mikepenz** [comment]
@tkirshboim thanks for the report. I will have a look.

**Comment by mikepenz** [comment]
Thanks for the report. this got fixed in v8.1.1
